packages/device_info_plus/device_info_plus/lib/src/model/android_device_info.dart

```diff
@@ -116,38 +117,10 @@ class AndroidDeviceInfo implements BaseDeviceInfo {
   /// Information about the current android display.
   final AndroidDisplayMetrics displayMetrics;
 
-  /// Serializes [AndroidDeviceInfo] to map.
-  @Deprecated('[toMap] method will be discontinued')
-  @override
-  Map<String, dynamic> toMap() {
-    return {
-      'id': id,
```

Comment:
The original `toMap` disappears and now instead the `data` returned by the platform channel is returned. Rather than re-creating the map out of the class data, we return the same data the Android code created, which should be 100% serializable.